EXAMPLES.md

```diff
@@ -22,11 +22,11 @@ Before you can start using the Unity Buy SDK, you need:
 
 - a Shopify store with at least one product
 - [a storefront access token for your app](https://help.shopify.com/api/storefront-api/getting-started#obtaining-a-storefront-access-token)
-- [to install the Unity Buy SDK into your Unity project](https://github.com/shopify/unity-buy-sdk#using-the-unity-buy-sdk-in-unity)
+- [to install the Unity Buy SDK into your Unity project](README.md#using-the-unity-buy-sdk-in-unity)
 
-## Using the SDK on iOS
+## Supported build targets
```

Comment:
**Supported build targets** and **Build Target Requirements** makes me think I'll see information like:

1. What OS does the Unity SDK support publishing to
2. What devices can the Unity SDK hit
etc.

But that link also has implementation details.